src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Extensions/ConfigurationBuilderExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System.IO;
+using Microsoft.Extensions.Configuration;
+
+
+namespace Altinn.App.PlatformServices.Extensions
+{
+    public static class ConfigurationBuilderExtensions
+    {
+        public static void LoadAppConfig(this IConfigurationBuilder builder)
+        {
+            string basePath = Directory.GetParent(Directory.GetCurrentDirectory()).FullName;
+
+            if (basePath == "/")
+            {                
+                builder.AddEnvironmentVariables();
```

Comment:
testing without AddEnvironmentVariables

Reply:
Does not work without explicitly adding the environment variables.. Not sure how to place this in startup? We don't have the same configuration builder object there. 
Also, it is loaded in program.cs for all the platform components, so what is the argument for moving this to startup for the app?

Reply:
It was working fine before this PR. I want to know why this PR changes this.

Reply:
Before this pr we didnt call  webBuilder.ConfigureAppConfiguration in Program.cs.. seems this sets back some default behavior?

Reply:
moved it to program.

Reply:
The documentation for `ConfigureAppConfiguration `
> This can be called multiple times and the results will be additive.

Reply:
Without explicitly using AddEnvVars this is the error message we get when starting an app running in a docker container. 
I can only assume this is because it isn't able to get the env variables related the jwtcookieHandler requires.

```
System.InvalidOperationException: IDX20803: Unable to obtain configuration from: 'System.String'.
 ---> System.ArgumentException: IDX20108: The address specified 'System.String' is not valid as per HTTPS scheme. Please specify an https address for security reasons. If you want to test with http address, set the RequireHttps property  on IDocumentRetriever to false. (Parameter 'address')
   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)
   at Microsoft.IdentityModel.Protocols.OpenIdConnect.OpenIdConnectConfigurationRetriever.GetAsync(String address, IDocumentRetriever retriever, CancellationToken cancel)
   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)
   --- End of inner exception stack trace ---
   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)
   at AltinnCore.Authentication.JwtCookie.JwtCookieHandler.HandleAuthenticateAsync()
   at AltinnCore.Authentication.JwtCookie.JwtCookieHandler.HandleAuthenticateAsync()
   at Microsoft.AspNetCore.Authentication.AuthenticationHandler`1.AuthenticateAsync()
   at Microsoft.AspNetCore.Authentication.AuthenticationService.AuthenticateAsync(HttpContext context, String scheme)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
?[40m?[32minfo?[39m?[22m?[49m: Microsoft.AspNetCore.Hosting.Diagnostics[2]
```